.gitattributes

```diff
@@ -0,0 +1 @@
+pnpm-lock.yaml linguist-generated
```

Comment:
nit: I figured GitHub would automatically include this path, nothing wrong with repeating it tho